invenio_records_lom/templates/invenio_records_lom/search.html

```diff
@@ -0,0 +1,86 @@
+{#
+  Copyright (C) 2022 Graz University of Technology
+
+  invenio-records-marc21 is free software; you can redistribute it and/or modify
+  it under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+
+{%- extends config.LOM_BASE_TEMPLATE %}
+
+
+{%- block javascript %}
+{{ super() }}
+{{ webpack['invenio-records-lom-search.js'] }}
+{%- endblock %}
+
+{%- block page_body %}
+
+<div data-invenio-search-config='{
```

Comment:
`invenio` has moved away from coding the `search-config` directly into html-files, newer html-templates have this inject (see, for example, [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/users_ui/templates/semantic-ui/invenio_app_rdm/users/requests.html#L21))

might be better to do it that way too...

Reply:
yeah i saw that. now it is harder to find the real config in the invenio jungle. but yeah i think too it would be better to have it hardcoded in the python code to.

i will create a issue to refactor that afterwards